.clj-kondo/config.edn

```diff
@@ -355,7 +355,7 @@
     metabase.sync.field-values                                      field-values
     metabase.sync.interface                                         i
     metabase.sync.schedules                                         sync.schedules
-    metabase.sync.sync-metadata.fields.fetch-metadata               fetch-metadata
+    metabase.sync.sync-metadata.fields.our-metadata                 our-metadata
```

Comment:
```suggestion
    metabase.sync.sync-metadata.fields.our-metadata                 fields.our-metadata
```

without this prefix, it's hard to know whether it's fields metadata or table metadata or db metadata

Reply:
agreed. we don't use this namespace much so the extra characters doesn't have much downside 
40df070bd39e3ed15ece0bd36c50fa1c80daef2a